pkg/domain/infra/abi/containers_linux.go

```diff
@@ -0,0 +1,35 @@
+package abi
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/containers/podman/v4/pkg/domain/entities"
+)
+
+func checkExecPreserveFDs(options entities.ExecOptions) error {
```

Comment:
Seems like this could be consolidated into a `containers_unix.go`, with just a per-OS constant for where the FDs live?

Reply:
I like this idea - I will re-work this part. I still suspect this test is redundant though :).